wpilibc/src/main/native/include/frc/simulation/DifferentialDrivetrainSim.h

```diff
@@ -36,10 +36,17 @@ class DifferentialDrivetrainSim {
    * create the drivetrainPlant.
    * @param wheelRadiusMeters The radius of the wheels on the drivetrain, in
    * meters.
+   * @param measurementStdDevs Standard deviations for measurements, in the form [heading, left velocity,
+   *                           right velocity, left distance, right distance]^T. Can be omitted if no noise is desired.
+   *                           Gyro standard deviations of 0.0001 radians, velocity standard deviations of 0.05 m/s,
+   *                           and position measurement standard deviations of 0.005 meters are a reasonable starting
+   *                           point.
    */
   DifferentialDrivetrainSim(const LinearSystem<2, 2, 2>& plant,
                             units::meter_t trackWidth, DCMotor driveMotor,
-                            double gearingRatio, units::meter_t wheelRadius);
+                            double gearingRatio, units::meter_t wheelRadius,
+                            const std::array<double, 5>& measurementStdDevs =
+                      std::array<double, 5>{});
```

Comment:
```suggestion
                      std::array<double, 5>{0.0, 0.0, 0.0, 0.0, 0.0});
```
Not zeroing these out causes UB.